package.json

```diff
@@ -22,7 +22,7 @@
   "scripts": {
     "clean": "rm -rf lib && bun run clean-test-temp",
     "clean-test-temp": "rm -rf src/commands/components/temp src/commands/components/init/temp",
-    "copy": "copyfiles --error 'templates/**' 'lib' && copyfiles --error --up 1 'src/run.cmd' 'lib'",
+    "copy": "copyfiles --error --all --exclude='.* !(|.spectral)' 'templates/**' 'lib' && copyfiles --error --up 1 'src/run.cmd' 'lib'",
```

Comment:
@awburgard probably not necessary now but marking this as something we may need to re-evaluate. This wasn't working as expected last time we touched this.